files/zh-cn/web/api/webvr_api/concepts/index.md

```diff
@@ -62,9 +62,9 @@ There are two main types of setup, mobile or computer-connected. Their minimum h
 **主要有两种创建 VR 环境的类型，手机或者是 PC。以下是实现这两种环境所需要的最少的硬件支持：**
 
 - Mobile: A Head-mounted display (HMD) is created using a smartphone — which acts as the VR display — mounted in a VR mount such as Google Cardboard, which contains the required lenses to provide stereoscopic vision of what is projected on the mobile screen.
-- **手机：通过使用一部智能手机可以营造一部头显设备（HMD）-- 扮演 VR 显示器的角色 -- 安装在一个像谷歌 CARDBOARD 那样的 VR 框架中，其中包含了必需要有的透镜，用以提供投射在手机屏幕上的立体视觉效果。【K】**![Mobile based VR setup](mobilebasedvrsetup.png)
+- **手机：通过使用一部智能手机可以营造一部头显设备（HMD）-- 扮演 VR 显示器的角色 — 安装在一个像谷歌 CARDBOARD 那样的 VR 框架中，其中包含了必需要有的透镜，用以提供投射在手机屏幕上的立体视觉效果。【K】**![Mobile based VR setup](mobilebasedvrsetup.png)
```

Comment:
This document has mixed English and zh-CN translation. I would create a PR to fix this.